crud.py

```diff
@@ -0,0 +1,140 @@
+import csv
+import os.path
+import logger as lg
+
+
+db_file_name = ''
+db = []
+global_id = 0  # id для добавления пользователей
+
+
+def init_data_base(file_name='db.csv'):
+    global global_id
+    global db
+    global db_file_name
+    db_file_name = file_name
+    db.clear()
+    if os.path.exists(db_file_name):
+        with open(db_file_name, 'r', newline='') as csv_file:
+            reader = csv.reader(csv_file)
+            for row in reader:
+                if(row[0] != 'ID'):
+                    db.append(row)
+                    if(int(row[0]) > global_id):
+                        global_id = int(row[0])
+    else:
+        open(db_file_name, 'w', newline='').close()
+
+
+def create(name='', surname='', number='', email=''):
+    global global_id
+    global db
+    global db_file_name
+    if(name == ''):
+        print("ALARM NO NAME SPECIFIED!!!!!1111")
+        return
+    if(surname == ''):
+        print("ALARM NO SURNAME SPECIFIED!!!!!1111")
+        return
+    if(number == ''):
```

Comment:
Почему вы везде ставите скобки в условиях? Случаи, где нужны скобки, можно по пальцам пересчитать.
```suggestion
    if number == '':
```